### DISCUSSION TITLE
Can't run Run.Bat or Install.cpl

### DISCUSSION THREAD

**Discussion by nyx1047** [LSPosed/MagiskOnWSALocal#543] [Q&A]
![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/133978771/b8b4000e-41cc-47e9-b008-88606a523a38)
Am I doing this wrong? I'm running everything on Ubuntu Terminal on Windows

**Comment by Howard20181** [comment]
Don't run it in WSL.
